### PR TITLE
Add documentation to update ARC with prometheus CRDs needed by actions metrics server

### PR DESCRIPTION
**Pull request by dhawalseth**
Deploy prometheus CRDs needed by actions metrics server to fix the following upgrade issue:
```
upgrade.go:142: [debug] preparing upgrade for actions-runner-controller
Error: UPGRADE FAILED: resource mapping not found for name: "arc-service-monitor" namespace: "actions-runner-system" from "": no matches for kind "ServiceMonitor" in version "monitoring.coreos.com/v1"
ensure CRDs are installed first
helm.go:84: [debug] resource mapping not found for name: "arc-service-monitor" namespace: "actions-runner-system" from "": no matches for kind "ServiceMonitor" in version "monitoring.coreos.com/v1"
ensure CRDs are installed first
```

### CODE REVIEW

File: charts/actions-runner-controller/docs/UPGRADING.md
Comment (mumoshu) [inline charts/actions-runner-controller/docs/UPGRADING.md]: I believe these Prometheus-operator-related CRDs are required for everything related to `serviceMonitor`. It might not be specific to chart 0.22.0+ or actionsmetrics-server.

Thats said, it would great to rephrase this a bit, like:

```suggestion
In case you're going to create prometheus-operator `ServiceMonitor` resources via the chart, you'd need to deploy prometheus-operator-related CRDs as well.

For example, you can use the `appscode/prometheus-operator-crds` to deploy those CRDs with Helm:
```

**Review by mumoshu — DISMISSED** [review]
LGTM. Thanks a lot for your contribution @dhawalseth!

Hey @Link- Perhaps you might want this to be reviewed by the documentation team before merging?

**Comment by dhawalseth** [comment]
@Link- can you please check it out. thanks!

File: charts/actions-runner-controller/docs/UPGRADING.md
Comment (TingluoHuang) [inline charts/actions-runner-controller/docs/UPGRADING.md]: which chart might be better? `prometheus-community/prometheus-operator-crds` or `appscode/prometheus-operator-crds`

Comment (Link-) [reply charts/actions-runner-controller/docs/UPGRADING.md]: @dhawalseth - [prometheus.io](http://prometheus.io/) holds no opinion (as far as we're aware) on how to install directly into the Kubernetes cluster. As such, we think it's best if we leave this decision for our end users to make.

Is there a particular reason you're recommending this specific solution?

File: charts/actions-runner-controller/docs/UPGRADING.md
Comment (Link-) [inline charts/actions-runner-controller/docs/UPGRADING.md]: This is helpful, we can keep this.

File: charts/actions-runner-controller/docs/UPGRADING.md
Comment (dhawalseth) [reply charts/actions-runner-controller/docs/UPGRADING.md]: No particular reason. we can leave this up to the end users.

**Review by TingluoHuang — APPROVED** [review]

**Pull request merged**
